lib/iris/fileformats/grib/__init__.py

```diff
@@ -1053,38 +828,24 @@ def save_pairs_from_cube(cube):
         yield (slice2D, grib_message)
 
 
-def as_messages(cube):
-    """
-    .. deprecated:: 1.10
```

Comment:
What is the process now that `as_messages` and `as_pairs` were deprecated in Iris 1.10. Are we ready to completely remove them from the codebase?